R/estimate_vaccineff.R

```diff
@@ -122,20 +122,40 @@ summary.vaccineff <- function(object, ...) {
   stopifnot("Input must be an object of class 'vaccineff'" =
       checkmate::test_class(object, "vaccineff")
   )
-  cat(
-    sprintf("Vaccine Effectiveness at %i days computed as VE = 1 - HR:\n",
-            object$at)
+  summ <- list(
+    ve = object$ve,
+    at = object$at,
+    p_value_schoenfeld = object$cox_model$p_value
+  )
+  class(summ) <- "summary.vaccineff"
+  return(summ)
+}
+
+#' @title Print summary of VE Results
+#'
+#' @description Print summary of `vaccineff` object.
+#'
+#' @param x Object of the class `summary.vaccineff`.
+#' @param ... Additional arguments passed to other functions.
+#' @return Summary of the results from `estimate_vaccineff`.
+#' @export
+
+print.summary.vaccineff <- function(x, ...) {
+  message(
```

Comment:
This function can use `print()` (or `writeLines()` or equivalent function) to print to console and does not need to use `message()`. Because it is a `print()` function the user will expect it to print to the console. It is only in other methods, e.g. `summary()` where you will want to use `message()` over `print()` or `cat()` as the user will not expect side effects (e.g. printing to the console) and therefore may want to silence it with `suppressMessages()`.

Reply:
Thanks for the comment. I misunderstood your observation during the full package review. But now it makes full sense. I solved it in 062d6a2b55cd2dc08d2c435871cab32d6c9144d1